security/Security.php

```diff
@@ -95,7 +95,7 @@ class Security extends Controller {
 	 *
 	 * @var array|string
```

Comment:
I’ve got no idea, but if it’s an empty string then setting the following throws a type mismatch exception in `Config`:

```yml
Security:
  default_message_set:
    'default': 'test'
```

Leaving it undefined seems to work fine with both a string and array though, so I’ll make that change :)